health-services/project-factory/CHANGELOG.md

```
@@ -14,3 +14,9 @@
    1. Timeline integration for workflow of campaign.
    2. Call user, facility and boundary generate when boundaries changed in campaign update flow
    3. Generate target template based on delivery conditions changed to anything from default.
+
+## 0.3.0 - 2024-12-03
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)